From 826c6b09ae2801ee331962cc9f4cbc25842e50d2 Mon Sep 17 00:00:00 2001
From: Jose Luis Rivero <jrivero@osrfoundation.org>
Date: Wed, 12 Aug 2015 02:21:46 +0200
Subject: [PATCH 1/2] Patch name to release -6 version

---
 debian/control.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control.em b/debian/control.em
index 175af20..857e836 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -8,7 +8,7 @@ Build-Depends: debhelper (>= @(debhelper_version).0.0), @(', '.join(BuildDepends
 Homepage: @(Homepage)
 Standards-Version: 3.9.2
 
-Package: @(Package)
+Package: @(Package.replace('gazebo-','gazebo6-'))
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends))
 Description: @(Description)
-- 
1.9.1

